kratos/utilities/geometry_utilities.cpp

```diff
@@ -518,7 +518,7 @@ void GeometryUtils::ShapeFunctionsSecondDerivativesTransformOnAllIntegrationPoin
 /***********************************************************************************/
 
 void GeometryUtils::ShapeFunctionsSecondDerivativesTransformOnIntegrationPoint(
-      Matrix DN_DX,
+      const Matrix DN_DX,
```

Comment:
you are passing this by copy. is it intentional?

Reply:
No, I think we can pass it as a reference